_sass/screen.scss

```diff
@@ -5,6 +5,18 @@ body {
   font-size: $base-font-size;
 }
 
+p {
+  max-width: 40em;
```

Comment:
I think if the intent here is to cap characters per line we should use `max-width:66ch`